src/protocol/organization.rs

```diff
@@ -175,6 +176,53 @@ impl OrganizationConfigReq {
     }
 }
 
+#[pyclass]
+#[derive(Debug, Clone)]
+pub(crate) struct ActiveUsersLimit(pub types::ActiveUsersLimit);
+
+#[pymethods]
+impl ActiveUsersLimit {
+    #[classmethod]
+    #[pyo3(name = "LimitedTo")]
+    fn limited_to(_cls: &PyType, user_count_limit: u64) -> Self {
+        Self(types::ActiveUsersLimit::LimitedTo(user_count_limit))
+    }
+
+    #[classattr]
+    #[pyo3(name = "NO_LIMIT")]
+    fn no_limit() -> &'static PyObject {
+        lazy_static::lazy_static! {
+            static ref VALUE: PyObject = {
+                Python::with_gil(|py| {
+                    ActiveUsersLimit(types::ActiveUsersLimit::NoLimit)
+                        .into_py(py)
+                })
+            };
+        };
+
+        &VALUE
+    }
+
+    #[classmethod]
+    #[pyo3(name = "FromOptionalInt")]
+    fn from_optional_int<'py>(cls: &'py PyType, py: Python<'py>, count: Option<u64>) -> &'py PyAny {
```

Comment:
```suggestion
    fn from_optional_int<'py>(cls: &'py PyType, py: Python<'py>, count: Option<u64>) -> &'py Self {
```

That way it would allow a more friendly typing for it